### PR TITLE
preparing for moving lesson to sphinx

### PR DESCRIPTION
**Pull request by bast**
Work with @samumantha

### CODE REVIEW

**Comment by bast** [comment]
I will adjust some more boxes and then un-draft hopefully later today. Later we can do all the polishing in follow-up PRs.

**Comment by bast** [comment]
Ready for review. After this is in we can fix svg, then merge the two similar episodes into one, then other issues and the draft PR.

File: content/conf.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+#    #'sphinx': ('https://www.sphinx-doc.org/', None),
+#    #'numpy': ('https://numpy.org/doc/stable/', None),
+#    #'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
+#    #'pandas': ('https://pandas.pydata.org/docs/', None),
+#    #'matplotlib': ('https://matplotlib.org/', None),
+#    'seaborn': ('https://seaborn.pydata.org/', None),
+# }
+
+def setup(app):
+    app.add_css_file("style.css")
```
Comment (rkdarst) [inline content/conf.py:106]: I reviewed everything but have some questions about this.  Is this style new for this lesson, or copied from another one?

By now we could add colors to sphinx-lesson directly.  Also, at least to my eye the colors don't look like the best and are somewhat inconsistent (e.g. objectives is orange, questions right above isn't.)  This isn't in sphinx-lesson-template.  I was wondering what's the strategy here?

Still, I would say it is better to merge and proceed and then we can figure out about colors later.  But will leave the merge to you.

**Review by rkdarst — APPROVED** [review]
I have questions about the colors and custom style, but we can handle that later to not hold back progress.

File: content/conf.py (hunk 1 above)
Comment (bast) [reply content/conf.py:106]: The colors have been copied from the data visualization lesson. Happy to change this and we can move it more upstream but I just wanted something else than only blue. So I consider this color choice only temporary until this moves more upstream.

**Comment by samumantha** [comment]
Sorry, was too late and just reviewing. Thanks for merging! :)

**Pull request merged**